hippod/app/assets/js/helper.js

```diff
@@ -158,9 +158,7 @@ function secondsToRelativeLifetime(seconds) {
     }
     if (seconds < 31104000) // 12 * 30 * 24 * 60 * 60
     {
-        return Math.ceil(seconds / (60 * 60 * 24 * 30)) + " month left (" +
-                 prev_date.getUTCDate() + "-" +  (prev_date.getUTCMonth() + 1) + "-" +
-                     prev_date.getUTCFullYear() + ")";
+        return Math.ceil(seconds / (60 * 60 * 24 * 30)) + " month left"
```

Comment:
no, don't calculate this here, JS is not that performant to detect this at parse time, just precalcuate this, save in an additional variable, name the variable accordingly and add a comment in the form of // 60 * 60 * 24 * 30